server/services/nl.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Language Model client."""
+
+from google.cloud import storage
+from sentence_transformers import SentenceTransformer
+from sentence_transformers.util import semantic_search
+
+import os
+import torch
+from datasets import load_dataset
+import logging
+
+BUILDS = ['demographics300', 'uncurated3000']
```

Comment:
As a note, looks like we are only using BUILD, so could load only that or leave TODO.

Reply:
Also, should we set BUILD as a config?